### PR TITLE
Mini cart popup

### PR DESCRIPTION
**Pull request by alisha331**

### CODE REVIEW

File: src/components/utilityNav/MiniCart.vue [hunk 1]
```diff
@@ -2,73 +2,128 @@
...
             return {
-                display: 'block'
+                display: 'none'
             }
         },
         components: {
             CartTile: () => import('../../views/CartTile')
         },
         computed: {
+            cartItems() {
```
Comment (rahimoddin) [inline src/components/utilityNav/MiniCart.vue:60]: Multiple cartItems

File: src/components/utilityNav/MiniCart.vue [hunk 2]
```diff
@@ -2,73 +2,128 @@
...
+            cartItem() {
+                return this.$store.getters['cart/items'];
+            },
+            currentItem() {
+                return this.$store.getters['cart/currentItem'];
+            },
             ...mapGetters('cart', {
-            count: 'itemCount',
-            cartItems: 'cartItems'
+                itemCount: 'itemCount',
```
Comment (rahimoddin) [inline src/components/utilityNav/MiniCart.vue:70]: duplicates

File: src/components/utilityNav/MiniCart.vue [hunk 3]
```diff
@@ -2,73 +2,128 @@
...
+                }
             },
             hideMiniCart() {
                 this.display = 'none'
-            }
+            },
+            viewCart(evt) {
+                evt.preventDefault();
+                this.display = 'none'
+                this.$router.push('/Cart');
```
Comment (rahimoddin) [inline src/components/utilityNav/MiniCart.vue:89]: change route case here use - /cart

File: src/components/utilityNav/MiniCart.vue [hunk 4]
```diff
@@ -2,73 +2,128 @@
...
+            },
+            checkout(evt) {
+                evt.preventDefault();
+                this.display = 'none'
+                this.$router.push('/Shipping');
```
Comment (rahimoddin) [inline src/components/utilityNav/MiniCart.vue:94]: change route case '/shipping'

File: src/store/modules/cart.js [hunk 5]
```diff
@@ -85,6 +107,18 @@ const actions =  {
     }
 }
 
+const cartHandler =  {
```
Comment (rahimoddin) [inline src/store/modules/cart.js:110]: A better solution can be added. Lets discuss this over call

File: src/components/utilityNav/MiniCart.vue (hunk 1 above)
Comment (alisha331) [reply src/components/utilityNav/MiniCart.vue:60]: Incorporated

**Pull request merged**